src/main/java/hu/eszter/bokkon/model/participants/Dice.java

```diff
@@ -1,4 +1,4 @@
-package hu.eszter.bokkon.model.tool;
+package hu.eszter.bokkon.model.participants;
```

Comment:
You could add a sanity check to the constructor to avoid creating a dice with 42 side dice